<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaAgregarPersonal</class>
 <widget class="QMainWindow" name="VentanaAgregarPersonal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VentanaAgregarPersonal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="AgregarPersonalWindow">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 12pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Agregar Personal</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_PrecioProd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>229</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cargo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_DescripProd">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Tel√©fono:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCargoPersonal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_CantidadProd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string> Correo:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_registrar">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;
background-color: rgb(0, 170, 0) ;</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Agregar.jpeg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_NombreProducto">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_volver">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;Arial&quot;;
background-color: rgb(0, 0, 255) ;</string>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineNomPersonal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCorreoPersonal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>270</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineTelPersonal">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
